src/wazuh_modules/wm_vuln_detector.h

```diff
@@ -36,35 +43,43 @@
 #define VU_ALERT_HEADER "[%s] (%s) %s"
 #define VU_ALERT_JSON "1:" VU_WM_NAME ":%s"
 #define VU_INV_OS     2
-#define VU_UBUNTU     "UBUNTU"
-#define VU_PRECISE    "PRECISE"
-#define VU_TRUSTY     "TRUSTY"
-#define VU_XENIAL     "XENIAL"
-#define VU_RHEL       "RED HAT"
-#define VU_CENTOS     "CENTOS"
-#define VU_RHEL5      "RHEL5"
-#define VU_RHEL6      "RHEL6"
-#define VU_RHEL7      "RHEL7"
 #define VU_MODERATE   "Moderate"
 #define VU_MEDIUM     "Medium"
 #define VU_HIGH       "High"
 #define VU_IMPORTANT  "Important"
 
 extern const wm_context WM_VULNDETECTOR_CONTEXT;
 
+static const char *vu_dist[] = {
```

Comment:
Declaring static (private) strings in a header has no sense. It produces warnings in all source files except which they are used in:

```
In file included from wazuh_modules/wmodules.h:64:0,
                 from wazuh_modules/main.c:12:
wazuh_modules/wm_vuln_detector.h:53:20: warning: ‘vu_dist’ defined but not used [-Wunused-variable]
 static const char *vu_dist[] = {
                    ^~~~~~~
```